test/web-auth/web-auth.test.js

```diff
@@ -1233,10 +1233,7 @@ describe('auth0.WebAuth', function() {
             hash: '#state=foo&access_token=VjubIMBmpgQ2W2&id_token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJodHRwczovL2F1dGgwLXRlc3RzLWxvY2suYXV0aDAuY29tLyIsImlhdCI6MTUwOTA0MDk4MiwiZXhwIjoxNTQwNTc2OTgyLCJhdWQiOiJpeGVPSEZoRDdOU1B4RVFLNkNGY3N3alVzYTVZa2NYUyIsInN1YiI6Impyb2NrZXRAZXhhbXBsZS5jb20iLCJHaXZlbk5hbWUiOiJKb2hubnkiLCJTdXJuYW1lIjoiUm9ja2V0IiwiRW1haWwiOiJqcm9ja2V0QGV4YW1wbGUuY29tIiwiUm9sZSI6WyJNYW5hZ2VyIiwiUHJvamVjdCBBZG1pbmlzdHJhdG9yIl19._JvcLjX308NtT16oegF2wFeOcdEYKM3DqX-V4POwIeg&token_type=Bearer&refresh_token=kajshdgfkasdjhgfas'
           },
           function(err, data) {
-            expect(err).to.be.eql({
-              error: 'invalid_token',
-              errorDescription: 'Algorithm HS256 is not supported. (Expected algs: [RS256])'
-            });
+            expect(err).to.be.eql({ any: 'error' });
```

Comment:
this is a breaking change. You ARE changing the error format.

Reply:
no I'm not. I'm just returning the original error from userInfo instead the validation error. I just mocked the userInfo error differently: https://github.com/auth0/auth0.js/pull/724/files#diff-548558e7a13abfcf8869090467ec6967R1228